scc/light_client/light_client.go

```diff
@@ -21,6 +22,9 @@ type LightClient struct {
 type Config struct {
 	Url     *url.URL
 	Genesis scc.Committee
+	// default values try only once per request with 10 seconds timeout
```

Comment:
nit: it seems a bit odd that the default "retry" wrapper is essentially disabling itself. If you only try once, there is not much of a retry.

Reply:
exactly, the idea for this was that one (as `LightClient` user) could disable the retry mechanism just by changing the config.

Reply:
after an offline discussion, the default behaviour will be similar to the timeout. when 0 the `newRetry` factory will retry 1024, which will most likely hit the timeout first. 
https://github.com/0xsoniclabs/sonic/pull/128/commits/1be35a75a4752a671ca398509b4002d0b30fac7f